CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [1.13.1](https://github.com/aws-observability/aws-rum-web/compare/v1.13.0...v1.13.1) (2022-02-24)
+
+
+### Bug Fixes
+
+* Set appropriate error codes in X-Ray segments ([#359](https://github.com/aws-observability/aws-rum-web/pull/359)) ([f6b7bd7](https://github.com/aws-observability/aws-rum-web/commit/f6b7bd744ec9e8413b93a2069c40b815e4ddc9ab))
```

Comment:
The correct commit message would be `Update version during release workflow only`, I don't think the commit is related to any x-ray changes